flavors/standard-EXASOL-7.1.0/flavor_customization/Dockerfile

```
@@ -20,7 +20,7 @@ COPY flavor_customization/packages/apt_get_packages /build_info/packages/flavor_
 RUN /scripts/install_scripts/install_via_apt.pl --file /build_info/packages/flavor_customization/apt_get_packages --with-versions
 
 COPY flavor_customization/packages/python3_pip_packages /build_info/packages/flavor_customization
-RUN /scripts/install_scripts/install_via_pip.pl --file /build_info/packages/flavor_customization/python3_pip_packages --python-binary python3.7 --with-versions --allow-no-version  --use-deprecated-legacy-resolver --ignore-installed
+RUN /scripts/install_scripts/install_via_pip.pl --file /build_info/packages/flavor_customization/python3_pip_packages --python-binary python3.8 --with-versions --allow-no-version  --use-deprecated-legacy-resolver --ignore-installed
```

```suggestion
RUN /scripts/install_scripts/install_via_pip.pl --file /build_info/packages/flavor_customization/python3_pip_packages --python-binary python3.8 --with-versions --allow-no-version
```
in another place you also removed this

---

Good catch, I actually forgot this